client/packages/common/src/intl/locales/en/replenishment.json

```diff
@@ -83,6 +83,7 @@
   "messages.inbound-shipment-created-on": "Inbound shipment created on {{date}}",
   "messages.internal-order-created-on": "Internal order created on {{date}}",
   "messages.off-hold-confirmation": "This will re-enable status changes",
+  "messages.cant-send-order": "Cannot change the status because there are no lines or because there are only placeholder lines",
```

Comment:
Nit but could it be a more specific message... i.e. Cannot send order/cannot send empty order or something? "Change status" feels vague/not quite right to me from a "send" button? 🤷‍♀️ ~